examples/demo.js

```diff
@@ -32,6 +32,9 @@ window.onload = function () {
 
   $(window).on('scroll', scrollFunction);
 
+  /**
+   * @description Method to toggle the scroll-up arro
```

Comment:
Why you used a single line comment on line 18 and multi-line comment on line 36?
Please read ``How to write good comments in JS?`` guides and find out when to use which type of comment.

Reply:
Here, line 18 describes the code below it. line 36 contains special JSDoc comment which is used to describe a *Function* below it. A JSDoc comment is interpreted differently by IDEs. Example below. This is the only reason I used JSDoc for functions.

![jsdoc](https://user-images.githubusercontent.com/34770591/70550098-41709700-1b9b-11ea-8156-bdf2a2ab38c0.gif)